### ISSUE TITLE
Email addresses enclosed in `<` and `>` are stripped from comments

### ISSUE THREAD

**Issue by prupert** [Icinga/icingaweb2#3738]
Special chars such as `<` and `>` are not escaped in comments leading to them being invisible and possibly rendering a security vulnerability

Example:  add a comment with text:
```
Can you see <this>?
```

## Expected Behavior

HTML chars should be escaped, comment should be visible in Icingaweb2 as:
```
Can you see <this>?
```

## Current Behavior

HTML chars not escaped, comment is visible as:
```
Can you see ?
```

## Your Environment

* Icinga Web 2 version and modules (System - About): 2.6.2 with generictts, grafana, monitoring

**Comment by nilmerg** [comment]
Hi,

comments are already properly handled in terms of security. Though, not plainly escaped but fed to a HTML parser which strips forbidden tags. (such as `<this>`)

Please show us a real world example that gets stripped away. We may then see what we're able to do against it.

**Comment by prupert** [comment]
I noticed something was off when setting an acknowledgement comment with the following body:
```
Redacted long test including a quoted log line: "message subject from=<noreply+redacted@redacted.nl>"
```

We did get the full message from Icinga through our alerts commands. But the part `<noreply+redacted@redacted.nl>` got stripped away in Icingaweb2. I first though this was due to Icingaweb2 not html-escaping it and immediately thought: oooh security bug. Luckily, that is not the case. 

What reason was there for stripping html tags in stead of escaping them?

**Comment by nilmerg** [comment]
We support using HTML in comments. Though, only a subset of tags are allowed so that the use of things like `<script>` is prevented. That email addresses are stripped though is not intended.

**Comment by nilmerg** [comment]
Related to #3684

**Comment by nilmerg** [comment]
Fixed by #3814. Mail addresses are not stripped anymore but are transformed to mailto links.
